public/locales/nn/translation.json

```diff
@@ -22,7 +22,7 @@
     "beholdning-and-pensjonspoeng-forklart-andel-pensjonspoeng": "Pensjonspoeng",
     "beholdning-and-pensjonspoeng-forklart-lenke": "Les om gamle og nye reglar i folketrygda.",
     "beholdning-and-pensjonspoeng-forklart-pensjonspoeng-title": "Pensjonspoeng forklart:",
-    "beholdning-and-pensjonspoeng-forklart-pensjonspoeng-text": "Pensjonspoeng blir berekna av pensjonsgivande inntekt, først og fremst arbeidsinntekt og personinntekt frå næringsverksemd kvart år, frå og med det året du fyller 17, til og med det året du fyller 69. Pensjonspoeng blir brukt som utgangspunkt for å berekne alderspensjonen du får frå NAV når du pensjonerer deg.",
+    "beholdning-and-pensjonspoeng-forklart-pensjonspoeng-text": "Pensjonspoeng blir rekna av pensjonsgivande inntekt, først og fremst arbeidsinntekt og personinntekt frå næringsverksemd. For personer født til og med 1942 opptjenes pensjonspoeng frå og med det året ein fyller 17 til og med det året ein fyller 69. For personer født frå 1943 til og med 1962 opptjenes pensjonspoeng frå det året ein fyller 17 til og med det året ein fyller 75.",
```

Comment:
"For personer født" -> "For personar fødde"

Når det gjelder "opptjenes pensjonspoeng" tror jeg ikke man kan bruke slik passivform på nynorsk, men må bruke f.eks. "blir pensjonspoeng opptent".

Reply:
Jeg endrer